EIPS/eof-contract-size.md

```diff
@@ -0,0 +1,79 @@
+---
+eip: nnnn
```

Comment:
```suggestion
eip: 7830
```
Assigning next sequential EIP/ERC/RIP number.
Numbers are assigned by editors & associates.

Please also update the filename.